pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1221,40 +1221,59 @@ func Convert_v1_Firmware_To_related_apis(vmi *v1.VirtualMachineInstance, domain
 	}
 
 	if firmware.ACPI != nil {
-		path, err := getSlicMountedPath(vmi.Spec.Volumes, firmware.ACPI.SlicNameRef)
-		if err != nil {
-			log.Log.Object(vmi).Warningf("Failed to get supported path for Volume: %s", firmware.ACPI.SlicNameRef)
-			return err
+		// If ACPI is set, we should have either SLIC or MSDM set.
```

Comment:
Convert previous line to early return? Avoids indentation of the following checks.

Reply:
The function is handling several firmware fields, the ACPI being the last we can do a early return but future changes would require to be added before that. If the goal is to lower the indentation, I can break split it in a separate function instead